spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/ThrowableCollector.kt

```diff
@@ -10,7 +10,7 @@ class ThrowableCollector {
         if (this.throwable == null) {
             this.throwable = throwable
         } else {
-            (this.throwable as java.lang.Throwable).addSuppressed(throwable)
+            (this.throwable as? java.lang.Throwable)?.addSuppressed(throwable)
```

Comment:
This seems unneded since there is a branch `if (this.throwable == null)` before and all throwables are children of `java.lang.Throwable`